modules/nf-core/ensemblvep/filtervep/tests/tab.gz.config

```diff
@@ -6,8 +6,6 @@
 
 process {
 
-    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
-
     withName: ENSEMBLVEP_DOWNLOAD {
         ext.args = '--AUTO c --CONVERT --NO_BIOPERL --NO_HTSLIB --NO_TEST --NO_UPDATE'
```

Comment:
Is the `--CONVERT` flag still needed here? It's been possible to download a pre-bgzipped index for a while now.

Reply:
I'll try this rn

Reply:
it is not changing anything in the snapshot, so I'll remove it